tls/service_checks.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "agent_version": "6.12.0",
+        "integration": "tls",
+        "groups": ["name", "server_hostname", "server", "port"],
+        "check": "tls.can_connect",
+        "statuses": ["ok", "critical"],
+        "name": "Can Connect",
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to the monitored endpoint, or `OK` otherwise."
+    },
+    {
+        "agent_version": "6.12.0",
+        "integration": "tls",
+        "groups": ["name", "server_hostname", "server", "port"],
+        "check": "tls.version",
+        "statuses": ["ok", "critical"],
+        "name": "Allowed TLS/SSL version",
+        "description": "Returns `CRITICAL` if a connection is made with a protocol version that is not allowed, or `OK` otherwise."
```

Comment:
```suggestion
        "description": "Returns `CRITICAL` if a connection is made with a protocol version that is not allowed, otherwise returns `OK`."
```